docs/app/pages/components/components-sections/select/select/select.component.html

```diff
@@ -16,7 +17,9 @@
                 [clearButton]="clearButton"
                 [(recentOptions)]="recentOptions"
                 [autoCloseDropdown]="autoCloseDropdown"
-                [recentOptionsMaxCount]="recentOptionsMaxCount">
+                [recentOptionsMaxCount]="recentOptionsMaxCount"
+                [listboxAriaLabel]="listboxAriaLabel"
+                [ariaLabelledby]="'select-ux-select-input-label'">
```

Comment:
```suggestion
                ariaLabelledby="select-ux-select-input-label">
```
Don't use dynamic bindings for a static value.